hdf5.spec

```
@@ -589,6 +583,9 @@ done
 %endif
 
 %changelog
+* Wed Jul 22 2020 Maureen Jean <maureen.jean@intel.com> - 1.12.0-1.g07066a381e
```

Shouldn't include the git hash here as it has been dropped from the Release: https://github.com/daos-stack/hdf5/blob/fd43495f8ab8ee0248033b1a8e9bff3c151639da/hdf5.spec#L16

```suggestion
* Wed Jul 22 2020 Maureen Jean <maureen.jean@intel.com> - 1.12.0-1
```